shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -1110,21 +1110,19 @@ public void requestDartDeferredLibrary(int loadingUnitId) {
    * @param loadingUnitId The loadingUnitId is assigned during compile time by gen_snapshot and is
    *     automatically retrieved when loadLibrary() is called on a dart deferred library. This is
    *     used to identify which Dart deferred library the resolved correspond to.
-   * @param searchPaths An array of paths in which to look for valid dart shared libraries. This
-   *     supports paths within zipped apks as long as the apks are not compressed using the
-   *     `path/to/apk.apk!path/inside/apk/lib.so` format. Paths will be tried first to last and ends
-   *     when a library is sucessfully found. When the found library is invalid, no additional paths
-   *     will be attempted.
+   * @param sharedLibraryName File name of the .so file to be loaded, or if the file is not already
+   *     in LD_LIBRARY_PATH, the full path to the file. Note: .so files in the lib/<abi> directory
```

Comment:
Remove `Note`. Also, may want to replicate this info about lib/<abi> in the DeferredComponentManager docs as well.

Reply:
Updated, and added this to the DeferredComponentManager docs. Thanks for the review!